web/web/models/service_location.py

```diff
@@ -17,18 +17,37 @@
 class ServiceLocation(Model):
     __tablename__ = 'service_locations'
 
-    service_location_id = Column(db.String(64),
-                                 primary_key=True,
+    service_location_id = Column(db.Integer,
+                                 primary_key=True, 
+                                 autoincrement=True, 
                                  nullable=False)
+
+    alternate_service_location_id = Column(db.String(64), 
```

Comment:
`alternate` is confusing. It technically isn't an alternate ID, it's just a unique name, I guess in this case it just happens to be provided by the creator.

Maybe just service location `name`? Or `provider_id`?

Reply:
this was a discussion during the sprint meeting. Mayank recommended sticking with alternate id, and David recommended name. Let me know your preference and I will implement!

Reply:
Got it. If there was a discussion on it, then it's fine. Just go with what was decided.